app-crypt/moolticute/moolticute-0.9.10_beta.ebuild

```diff
@@ -4,18 +4,18 @@
 EAPI=6
 
 if [[ ${PV} == 9999* ]]; then
-	EGIT_REPO_URI="https://github.com/raoulh/moolticute.git"
+	EGIT_REPO_URI="https://github.com/mooltipass/moolticute.git"
 	inherit git-r3
 	KEYWORDS=""
 else
-	SRC_URI="https://github.com/raoulh/moolticute/archive/v${PV/_/-}.tar.gz -> ${P}.tar.gz"
+	SRC_URI="https://github.com/mooltipass/moolticute/archive/v${PV/_/-}.tar.gz -> ${P}.tar.gz"
 	KEYWORDS="~amd64 ~arm"
 fi
 
 inherit gnome2-utils qmake-utils udev
 
 DESCRIPTION="Mooltipass crossplatform daemon/tools"
-HOMEPAGE="https://github.com/raoulh/moolticute"
+HOMEPAGE="https://github.com/mooltipass/moolticute"
```

Comment:
Any word on the URI changes?

Reply:
The project became officially supported by the mooltipass organisation, so they moved it into the org.